packages/at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -539,6 +539,7 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
   /// commit-id to sync into the local keystore.
   Future<List<dynamic>> _getEntriesToSyncFromServer(
       int lastReceivedServerCommitId) async {
+    // Sync verb syntax has to be changed before removing these deprecations
```

Comment:
Can you do a PR in at_lookup to change SyncVerbBuilder so it has default values for the deprecated things; and then do another PR here to remove the deprecated params